grzyClothTool/Models/Drawable/GDrawable.cs

```diff
@@ -60,13 +60,38 @@ public string TypeName
         }
     }
 
+    private string _sexName;
+    public string SexName
+    {
+        get
+        {
+            _sexName ??= EnumHelper.GetSex(Sex ? 1 : 0);
+            return _sexName;
+        }
+        set
+        {
+            _sexName = value;
+        }
+    }
+
     [JsonIgnore]
     public List<string> AvailableTypes => IsProp ? EnumHelper.GetPropTypeList() : EnumHelper.GetDrawableTypeList();
+    public static List<string> AvailableSex => EnumHelper.GetSexTypeList();
```

Comment:
add jsonIgnore for this

Basically JsonIgnore is used for stuff that are is needed in auto-saves jsons